rest-api-spec/src/main/resources/rest-api-spec/test/search.aggregation/370_multi_terms.yml

```diff
@@ -760,3 +760,75 @@ setup:
   - match: { aggregations.histo.buckets.2.key_as_string: "2022-03-25T00:00:00.000Z" }
   - match: { aggregations.histo.buckets.2.m_terms.buckets.0.key: [ "a", "127.0.0.1" ] }
   - match: { aggregations.histo.buckets.2.m_terms.buckets.1.key: [ "b", "127.0.0.1" ] }
+
+---
+"multi_terms bucket and multiple orders test":
+  - skip:
+      version: "- 2.0.99"
+      reason:  multi_terms aggregation is introduced in 2.1.0
+
+  - do:
+      bulk:
+        index: test_1
+        refresh: true
+        body:
+          - '{"index": {}}'
+          - '{"str": "a", "integer": 1, "double": 1234.5, "boolean": true}'
+          - '{"index": {}}'
+          - '{"str": "a", "integer": 2, "double": 5678.9, "boolean": true}'
+          - '{"index": {}}'
+          - '{"str": "a", "integer": 1, "double": 123.4, "boolean": false}'
+          - '{"index": {}}'
+          - '{"str": "a", "integer": 2, "double": 456.7, "boolean": false}'
+          - '{"index": {}}'
+          - '{"str": "b", "integer": 3, "double": 1234.5, "boolean": true}'
+          - '{"index": {}}'
+          - '{"str": "b", "integer": 4, "double": 5678.9, "boolean": true}'
+          - '{"index": {}}'
+          - '{"str": "b", "integer": 3, "double": 234.5, "boolean": false}'
+          - '{"index": {}}'
+          - '{"str": "b", "integer": 4, "double": 456.7, "boolean": false}'
+
+  - do:
+      search:
+        index: test_1
+        size: 0
+        body:
+          aggs:
+            m_terms:
+              multi_terms:
+                terms:
+                  - field: str
+                  - field: boolean
+                order:
+                  - max_int: desc
+                  - min_double: asc
```

Comment:
Shall we add a case for supporting size = 5?

Reply:
Sure!

Reply:
Yes, I will add one more case later.

Reply:
Test case added for the 'size' of multiple terms aggregation. @ankyhe